cmd/gossamer/config.go

```diff
@@ -162,8 +155,10 @@ func createInitConfig(ctx *cli.Context) (*dot.Config, error) {
 		return nil, fmt.Errorf("--%s must be %s", PruningFlag.Name, pruner.Archive)
 	}
 
-	if cfg.Global.RetainBlocks < dev.DefaultRetainBlocks {
-		return nil, fmt.Errorf("--%s cannot be less than %d", RetainBlockNumberFlag.Name, dev.DefaultRetainBlocks)
+	const minRetainBlocks = uint32(512)
```

Comment:
why are we using this `const` now?

Reply:
Polkadot, kusama, and westend all have a defaultRetainBlocks value of uint32(512). However, westend dev and westend local do not have this value set. This was a change made before I took over the PR but I imagine that this seemed like the most straightforward was to set the minRetainBlocks value. However, the name of the chain being used is stored in the context, so I could do a switch and set it based on what chain we were using. 

If this is preferred let me know, and then we also need to decide the minimum retained blocks for chains where this default value isn't present.

Reply:
This works for me.